cicd/README.md

```diff
@@ -37,6 +37,9 @@ optional arguments:
                         path to credentials file
   -i CREDENTIALS_INFO, --credentials_info CREDENTIALS_INFO
                         service account credentials info (alternative to file)
+  -e CREDENTIALS_ENV, --credentials_env CREDENTIALS_ENV
+                        service account credentials info stored in environment
+                        variable
```

Comment:
do we need to specify what happens if multiple creds are set? e.g. file and env are set, or info and env, which takes precedence?

Reply:
Resolved in last push